src/main/java/org/jenkinsci/plugins/github/util/JobInfoHelpers.java

```diff
@@ -33,49 +35,49 @@ private JobInfoHelpers() {
      *
      * @return predicate with true on apply if job contains trigger of given class
      */
-    public static Predicate<Job> withTrigger(final Class<? extends Trigger> clazz) {
-        return new Predicate<Job>() {
-            public boolean apply(Job job) {
-                return triggerFrom(job, clazz) != null;
+    public static <ITEM extends Item> Predicate<ITEM> withTrigger(final Class<? extends Trigger> clazz) {
```

Comment:
The type gets erased. It may cause compiler errors if compiling against the newer API but the erased type is still `Predicate`